.github/workflows/ci-pytorch-test-full.yml

```diff
@@ -121,11 +121,15 @@ jobs:
         pip list
       shell: bash
 
-    - name: DocTests
+    - name: DocTests PL
       if: ${{ (steps.skip.outputs.continue == '1') }}
       working-directory: ./src
       run: pytest pytorch_lightning --cov=pytorch_lightning
 
+    - name: DocTests actions
```

Comment:
This should be a separate job, instead of putting it in the pl job

Reply:
Good point, then maybe add it to the pkg install as this module is used for package creation...

Reply:
moved it to testing pkgs install